manifests/o/Oracle/JDK/21/21.0.1/Oracle.JDK.21.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Oracle.JDK.21
+PackageVersion: 21.0.1
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: wix
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: uninstallPrevious

```

```suggestion
InstallerSwitches:
  InstallLocation: INSTALLDIR=<INSTALLPATH>
UpgradeBehavior: uninstallPrevious
```
Should carry over metadata from previous versions